packages/web3-core-method/src/methods/txpool/GetTxpoolContentMethod.js

```diff
@@ -0,0 +1,36 @@
+/*
+    This file is part of web3.js.
+
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file GetTxpoolContentMethod.js
+ * @author Prince Sinha <sinhaprince013@gmail.com>
+ * @date 2019
+ */
+
+import AbstractMethod from '../../../lib/methods/AbstractMethod';
+
+export default class GetTxpoolContentMethod extends AbstractMethod {
```

Comment:
I would rename this method to: ``ContentMethod`` because it's already in the ``txpool`` folder :)

Reply:
It should be ContentMethod or GetContentMethod? @nivida

Reply:
Done @nivida. I replaced StatusMethod & InspectMethod too.